### ISSUE TITLE
Head and neck skin lesions

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9983]
**Preferred term label:**
Head and neck skin lesions

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Acne keloidalis nuchae

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
perhaps it is better to make a term called Acne keloidalis nuchae because "head and neck skin lesions" is quite nonspecific?

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000856
- label: Head and neck distribution
- parents: [Localized (HP:0012838)](https://hpo.jax.org/app/browse/term/HP:0012838)
- pmid: 
- def: Applies to an abnormality that is situated in (or predominant in) the head and neck region.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9983
- synonyms:
